ext/openssl/php_openssl.h

```diff
@@ -66,6 +66,12 @@ extern zend_module_entry openssl_module_entry;
 
 #include <openssl/err.h>
 
+#ifdef PHP_WIN32
```

Comment:
This might need a case to handle GCC as well, see:
https://github.com/php/php-src/blob/master/ext/libxml/php_libxml.h#L30